tasks/windows.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Manage the state of Windows services (without a puppet agent)",
+  "input_method": "powershell",
+  "parameters": {
+    "action": {
+      "description": "The operation (start, stop, restart, enable_automatic, enable_manual, disable, status) to perform on the service",
+      "type": "Enum[start, stop, restart, enable_automatic, enable_manual, disable, status]"
+    },
+    "name": {
+      "description": "The name of the service to operate on.",
```

Comment:
May be worth adding a  note here that the service name is Short version of the name, not the displayname.